src/SDKs/DataLake.Analytics/Management.DataLake.Analytics/Microsoft.Azure.Management.DataLake.Analytics.csproj

```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.DataLake.Analytics</PackageId>
     <Description>Provides Data Lake Analytics account, job and catalog management capabilities for Microsoft Azure.</Description>
-    <Version>3.5.0-preview</Version>
```

Comment:
I believe 2 SDKs were modified here, please bump versions in boths csproj files

Reply:
No, only DataLakeAnalyticsJobManagementClient is modified. Some changes are automatically generated without any change to the API Spec.